packages/shared/src/index.js

```diff
@@ -2,8 +2,9 @@ export { Button } from './components/Button';
 export { Places } from '@booking-ui/places';
 export { Geolocation } from '@booking-ui/geolocation';
 export { MessagingSystem, TextBox } from '@booking-ui/messaging-system';
-export { Demo } from '@booking-ui/search-engine';
+// export { Demo } from '@booking-ui/search-engine';
```

Comment:
That line was causing an error to start the project

Reply:
Can you ask the search engine team if they already fixed this? We should report this kind of issues to the squads since this code is not ours, let me know the resolution on this